src/main/scala/com/meetup/auth/MeetupConsumer.scala

```diff
@@ -116,23 +121,26 @@ class MeetupConsumerImpl(configuration: Configuration)(
     }
 
     val resp = opt.toOption
+    responseHandler(resp)
+  }
+
+  private[auth] val defaultGetResponseHandler: Option[Response] => Option[String] = defaultResponseHandler("GET")
+  private[auth] val defaultPostResponseHandler: Option[Response] => Option[String] = defaultResponseHandler("POST")
 
-    resp match {
+  private[auth] def defaultResponseHandler(methodType: String)(responseOpt: Option[Response]) = {
+    responseOpt match {
       case None => None
       case Some(r) if r.getStatusCode >= 400 =>
-        log.error(
-          s"received non-200 response from POST: $hostAndPath\n" +
-            s"with contentBody:\n    $contentBody\n" +
-            s"and headers:\n    $headers\n" +
-            s"and http status code: ${r.getStatusCode}\n" +
-            s"and body: ${r.getResponseBody}\n"
-        )
+        log.error(s"received non-200 response from $methodType\n" +
+          s"and http status code: ${r.getStatusCode}\n" +
+          s"and body: ${r.getResponseBody}\n")
```

Comment:
Could you please remove line breaks from the log? It causes log aggregators to record a single log statement as multiple log records

Reply:
Also you don't log the request url and payload. I think we should log at least the request URL to be able to correlate errors with requests

Reply:
Done.